src/components/listBox/ListBoxReducers.ts

```diff
@@ -79,21 +86,53 @@ export const listBoxReducer = (state: IListBoxState = listBoxInitialState, actio
                 ...state,
                 active: 0,
                 selected: [],
-                items: [],
             };
         case SelectActions.toggle:
+            const items: IItemBoxProps[] = _.map(state.items, (item: IItemBoxProps) => {
+                const hidden: boolean = _.some(state.selected, (selectedValue: string) => selectedValue === item.value);
+                return {
+                    ...item,
+                    hidden,
+                };
+            });
             return {
                 ...state,
+                items: items,

```

Comment:
`items,`

Reply:
```
items: state.items((item) => ({...item, hidden: _.contains(state.selected, item.value)}))  
```  
you can remove the `const items` logic altogether if you just write the items property this way